task_02/xieyu_wang/graphic.cpp

```diff
@@ -0,0 +1,128 @@
+#include <cstdlib>
+#include <fstream>
+#include <iostream>
+const int maxDots = 14;
+const int maxVecPerDot = 4;
+const int maxEdges = 18;
+// constexpr will be generator a error so cant use maybe completer's problem
```

Comment:
`constexpr` должно работать, проверь свой компилятор
```suggestion
constexpr int maxDots = 14;
constexpr int maxVecPerDot = 4;
constexpr int maxEdges = 18;
```

Reply:
Читай [Naming Convention](https://github.com/Friend-LGA/uni_course_cpp/tree/master/tasks#naming-convention)